src/main/scala/ru/org/codingteam/horta/protocol/jabber/MucMessageHandler.scala

```diff
@@ -24,7 +24,8 @@ class MucMessageHandler(locale: LocaleDefinition,
 
   val core = context.actorSelection("/user/core")
 
-  var participants = Map[String, Affinity](s"$roomJID/$nickname" -> User)
+  val participant = Protocol.Participant(s"$roomJID/$nickname", User, Participant) // TODO: Get own role and affiliation
```

Comment:
It means that currently the bot always thinks that he is `User` and `Participant` in any room he can enter. It is not always true, sometimes he can be `Moderator` or even `Owner`.